src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int leg;
+    private int base;

```

Comment:
It looks like you have only one 'base' property, but an isosceles trapezoid typically has two bases of different lengths. Consider adding another property for the second base.